app/utilities/supplementary_data_parser.py

```diff
@@ -67,25 +58,33 @@ class SupplementaryDataMetadataSchema(Schema, StripWhitespaceMixin):
 
     @validates_schema()
     def validate_dataset_and_survey_id(  # pylint: disable=unused-argument
```

Comment:
The function name should probably change since you also check "sds_schema_version" now.

Reply:
Instead, I moved the sds_schema_version validation in it's own function underneath, just because it makes the name look smaller in addition to the method name making sense 😅 and validation still passes.

Reply:
Good, that new function will require some unit testing (if it's not already tested).

Reply:
V minor - No worries if this has already been discussed, so feel free to ignore me 😆  but we now have 2 similar functions that validate things in the payload as follows: 

* existing `validate_dataset_and_survey_id()` which validates 2 things in the payload
* newly added `validate_sds_schema_version()` which validates 1 thing in the payload
 
would it be worth splitting `validate_dataset_and_survey_id` into separate validation functions, if we're making the validation modular? 

Personal preference, but I feel like it might make the most sense to keep all validation logic for this as one simple function (like it was originally in the PR I think?) and just rename it to something like `validate_payload()`?